src/main/java/org/example/calculator/model/Calculator.java

```diff
@@ -0,0 +1,47 @@
+package org.example.calculator.model;
+
+import java.util.Arrays;
+import java.util.function.Function;
+
+public class Calculator {
+
+    private static final String DEFAULT_DELIMITER = ",|:";
+    private static final String CUSTOM_DELIMITER_REGEX = "//(.)\n";
+    private static final String CUSTOM_OPERANDS_REGEX = "//.\n(.+)";
+    private final String string;
+
+    public Calculator(String string) {
+        this.string = string == null ? "" : string;
+    }
+
+    public int calculate() {
+        String delimiter = CalculatorStringUtils.extractDelimiterWithRegex(string, CUSTOM_DELIMITER_REGEX, DEFAULT_DELIMITER);
+        String operandString = CalculatorStringUtils.extractOperandStringWithRegex(string, CUSTOM_OPERANDS_REGEX);
+        String[] operands = splitOperandStringWithDelimiter(operandString, delimiter);
+
+        return getStringOperandsSum(operands);
+    }
+
+    private int getStringOperandsSum(String[] operands) {
+        return Arrays.stream(operands)
+                .map(emptyStringToZero())
```

Comment:
blank인 케이스들을 제외해버리면  `emptyStringToZero` 메서드를 따로 만들 필요도 없으면서 똑같은 동작을 할 수 있지 않을까요? 🙂
```suggestion
                .filter(operand -> !operand.isBlank())
```